index.css

```diff
@@ -284,4 +287,67 @@ body{
          display: none !important;
      }
  }
+  .hamburger {
+     color: #888;
+     cursor: pointer;
+     margin-top: 8px;
+     margin-bottom: 5px;
+     margin-left: 20px;
+     margin-right: 5px;
+     padding-top: 2px;
+     padding-bottom: 2px;
+     padding-left: 5px;
+     padding-right: 5px;
+     border-radius: 7px;
+ }
+ .cross {
+     color: #888;
+     cursor: pointer;
+     margin-top: 8px;
+     margin-bottom: 5px;
+     margin-left: 20px;
+     margin-right: 5px;
+     padding-top: 5px;
```

Comment:
Use relative values instead of `px`. Refer: https://www.w3schools.com/cssref/css_units.asp